evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -2705,7 +2705,7 @@ def test_get_component_input_logical_types():
         "cat_b": Boolean(),
         "cat_c": Boolean(),
         "EMAIL_ADDRESS_TO_DOMAIN(email)_gmail.com": Boolean(),
-        "IS_FREE_EMAIL_DOMAIN(email)_True": Boolean(),
+        "IS_FREE_EMAIL_DOMAIN(email)_1.0": Boolean(),
```

Comment:
Uh oh, I don't know if this is desired, right? @Cmancuso

Reply:
Yeah, I'm concerned about this as well, and I don't quite understand why it's happening. I spent some time digging into it and it seems to be stemming exclusively from scikit-learn. We correctly pass booleans into the encoder, but this came out as a float with no change on our end.